docs/wiki/99 - Developers/Automated Testing.md

```diff
@@ -0,0 +1,91 @@
+# Automated Testing
+
+
+## Overview
+We make use of TravisCI and various automated testing tools in order to make sure LORIS is clean, stable, and secure.
+
+We require that any pull request sent to the repository passes our entire suite of automated tests. If the code fails these tests, it cannot be merged as it is likely to break some functionality of the application.
+
+This guide provides a high-level overview of the tools we use for testing. It should help to demystify the process of understanding an automated test report and point the way to possible solutions by linking to the documentation fo the tools we use.
+
+Generally there are two categories of testing, **static** and **dynamic**. The following section breaks down the differences between the two and which tools are used for each.
+
+## Static Tests
+
+**Static tests** analyze the code without running it, checking for consistency between documentation and the source code.
```

Comment:
```suggestion
**Static tests** analyze the code without running it.
```

The part deleted is only one example of static analysis. Tools like phan analyze the code for things that are incorrect even if there's no documentation.

I also don't like the word "Static Tests" because "Static Analysis Tools" is more accurate, but rephrasing this whole section to use the correct terminology is beyond the scope of a GitHub edit suggestion..